sgt-back-end/index.jsx

```diff
@@ -0,0 +1,140 @@
+const express = require('express');
+const pg = require('pg');
+const app = express();
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:lfz@localhost/studentGradeTable'
+});
+
+app.use(express.json());
+
+app.get('/api/grades', (req, res, next) => {
+  const sql = `
+    select *
+      from "grades"
+  `;
+  db.query(sql)
+    .then(result => {
+      const grades = result.rows;
+      if (!grades) {
+        res.status(500).json({
+          error: 'We are currently unable to handle your reguest'
+        });
+      } else {
+        res.json(grades);
+      }
+    });
+});
+
+app.post('/api/grades', (req, res, next) => {
+  const sql = `
+insert into "grades" ("name", "course", "grade")
+values ($1, $2, $3)
+returning *
+`;
+  const params = [req.body.name, req.body.course, req.body.grade];
+  if (!req.body.name || !req.body.course || !req.body.grade) {
+    res.status(400).json({
+      error: 'Please provide a valid grade'
+    });
+  }
+  db.query(sql, params)
+    .then(result => {
+      const grade = result.rows[0];
+      if (grade) {
+        res.status(201).json({
+          success: 'Grade successfully added.'
+        });
+      } else {
+        res.json(grade);
+      }
```

Comment:
If an insert is successful and you included a `returning` clause in your sql, then you do not need this conditional.

Reply:
Also, the client should receive the grade object and not a message